arches/app/utils/data_management/resources/formats/format.py

```diff
@@ -308,7 +308,13 @@ def get_tiles(self, graph_id=None, resourceinstanceids=None, **kwargs):
             )
 
         if graph_id:
-            filters = {"resourceinstance__graph_id": graph_id}
+            filters = {
+                "resourceinstance__graph_id": graph_id,
+                # Get tiles only from the latest graph publication.
+                "resourceinstance__graph_publication": models.F(
+                    "resourceinstance__graph__publication"
```

Comment:
...doing the same thing here.

This avoids doing whack-a-mole inside rdffile.py